### DISCUSSION TITLE
[Script request] Immich

### DISCUSSION THREAD

**Discussion by jalvarezz13** [community-scripts/ProxmoxVE#414] [Request script]
### Application Name

Immich

### Website

https://immich.app/

### Description

Immich is an open-source self-hosted photo and video backup solution designed for personal and private cloud storage. It aims to offer users an alternative to cloud services like Google Photos, with features that include automatic media backup, AI-powered image tagging, and face recognition. Immich is lightweight, easy to deploy, and supports multiple devices.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Duplicate of: 
https://github.com/community-scripts/ProxmoxVE/discussions/43
